SEED-platform/seed
# Address PM Import Error

#### Any background context you want to provide?
Users are seeing 500 errors when trying to connect to portfolio manager.

#### What's this PR do?
As of right now it just adds extra error handling, because I can't reproduce the problem to investigate.  We'll need to get a screen share session to allow investigating further.
* Sort the user email addresses when creating a new organization

#### How should this be manually tested?
This should be pushed to a dev server, and a screen share should be set up to allow me to see a little more of the problem so I can figure out what we need to do to increase our robustness.

#### What are the relevant tickets?
#1654
